sql/derived_tables/item_ext.sql

```diff
@@ -45,13 +45,13 @@ SELECT
     items.effective_location_id,
     item_effective_location.name AS effective_location_name,
     json_extract_path_text(items.data, 'circulationNotes', 'descriptionOfPieces') AS description_of_pieces,
-    json_extract_path_text(items.data, 'status', 'date') AS status_date,
+    json_extract_path_text(items.data, 'status', 'date')::timestamptz AS status_date,
     json_extract_path_text(items.data, 'status', 'name') AS status_name,
     items.holdings_record_id,
     items.discovery_suppress,
-    json_extract_path_text(items.data, 'metadata', 'createdDate') AS created_date,
+    json_extract_path_text(items.data, 'metadata', 'createdDate')::timestamp AS created_date,
```

Comment:
I'm wondering about why this is just timestamp, instead of timestamptz. The [update to po_instance](https://github.com/folio-org/folio-analytics/pull/683/files) uses timestamptz for the same field.

Reply:
I have the same question. I went through past pull requests because I did a similar thing in the instance_ext derived table. Nassib recommended taht we use timestamptz for everything.

Reply:
I'm retracting my question here. #594 explains that `createdDate` should just be timestamp, so this looks okay. But I'll wait to review until there is an entry added to CHANGES.md.